src/components/Table/TableVirtualizedBody/TableVirtualizedBody.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ComponentProps, FC } from "react";
+import { VibeComponentProps } from "../../../types";
+import VirtualizedList from "../../VirtualizedList/VirtualizedList";
+import TableBody from "../TableBody/TableBody";
+
+export interface ITableVirtualizedBodyProps extends VibeComponentProps {
+  items: ComponentProps<typeof VirtualizedList>["items"];
+  itemRenderer: ComponentProps<typeof VirtualizedList>["itemRenderer"];
+}
+
+const TableVirtualizedBody: FC<ITableVirtualizedBodyProps> = ({ items, itemRenderer }) => {
+  return (
+    <TableBody>
+      <VirtualizedList items={items} itemRenderer={itemRenderer} getItemHeight={() => 40} layout="vertical" />
```

Comment:
in general our best practice right now it's to use useCallback when passing a function as prop to React component (and not actual doc element like div or span) - because we are assuming that we are not familiar with the component logic and it may result more not necessary renders

Reply:
The function is wrapped with `useCallback` inside `VirtualizedList`, so it might be redundant